src/cpp/tests/cpp/tests/vendor/catch.hpp

```diff
@@ -5965,7 +5965,7 @@ namespace Catch {
         void operator=( Version const& );
     };
 
-    extern Version libraryVersion;
+    inline Version libraryVersion();
```

Comment:
These changes were made to avoid a double-destruct crash in our `rsession` tests -- because we might load R packages that use Catch themselves, when closing the R session we can end up seeing this struct destructed twice (once by the version of Catch in the R package, and once by ourselves). I filed an associated bug on the Catch repository.